helm/cluster/templates/containerd_config.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "cluster.resource.name" $ }}-registry-configuration
```

Comment:
Not sure if this exact name is importnant, because the contents here is the entire containerd config, not just registry configuration (we just specify registry configuration in containerd config).